webui/react/config-overrides.js

```diff
@@ -14,9 +14,23 @@ const path = require('path');
 const webpack = require('webpack');
 const jestConfig = require('./jest.config');
 
-const IS_DEV = process.env.NODE_ENV === 'development';
+const IS_DEV = process.env.DET_NODE_ENV === 'development';
 const SERVER_ADDRESS = IS_DEV ? 'http://localhost:8080' : process.env.SERVER_ADDRESS;
 
+function customOverride(config, env) {
```

Comment:
**suggestion:** maybe call this `removeOptimizations` instead?

Reply:
I was thinking this would remain as a simple hook where we inject custom overrides where we have them? currently, it's just doing what you suggested tho :thinking:

Reply:
ah true, maybe just a one line comment somewhere below explaining that it's removing any webpack optimizations for a faster build.